test/test-auth.js

```diff
@@ -0,0 +1,208 @@
+global.DATABASE_URL = 'mongodb://localhost/jwt-auth-demo-test';
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const jwt = require('jsonwebtoken');
+
+const {app, runServer, closeServer} = require('../server');
+const {User} = require('../users');
+const {JWT_SECRET} = require('../config');
+
+const expect = chai.expect;
```

Comment:
NBD, but I think we've been mainly using should style syntax elsewhere. Will either need to explain this syntax or switch. My preference would be to consistently use `should`.